packages/accounts-2fa/package.js

```diff
@@ -0,0 +1,22 @@
+Package.describe({
+  version: '1.0.0',
+  summary:
+    'Package used to enable two factor authentication through OTP protocol',
+});
+
+Npm.depends({
+  'node-2fa': '2.0.3',
+  'qrcode-svg': '1.1.0',
+});
+
+Package.onUse(function(api) {
+  api.use(['accounts-base'], ['client', 'server']);
```

Comment:
I believe accounts-password should be here as well, AFAIK it's required as a dependency for this to work.

Is that not the case? If not, why?

Reply:
This package doesn't use any dependency from accounts-password... It is true that the developer needs to have both packages, but as they don't share direct dependencies, and they "communicate" through the Accounts object, I don't think it's necessary.

Reply:
Adding this package will add accounts-password automatically if you put it on api.use, that is why this is necessary.

This package doesn't work without accounts-password.